src/qililab/result/experiment_results.py

```diff
@@ -107,6 +116,25 @@ def get(self, qprogram: int | str = 0, measurement: int | str = 0) -> tuple[np.n
 
         return data, dims
 
+    def get_data(self, qprogram: int | str = 0, measurement: int | str = 0) -> np.ndarray:
+        """Retrieves only data for a specified quantum program and measurement.
+
+        Args:
+            qprogram (int | str, optional): The index or name of the quantum program. Defaults to 0.
+            measurement (int | str, optional): The index or name of the measurement. Defaults to 0.
+
+        Returns:
+            np.ndarray: The data array.
+        """
+        if isinstance(qprogram, int):
+            qprogram = f"QProgram_{qprogram}"
+        if isinstance(measurement, int):
+            measurement = f"Measurement_{measurement}"
+
+        data = self.data[qprogram, measurement][()]
+
+        return data
```

Comment:
This is duplicate code. There is the `get` method that returns data and dimensions. If you need only the data, one can do `data, _ = results.get()`.

Reply:
You are right, also right now is not even used, this is a leftover from some tests I did before. Thanks for pointing this out!